client.go

```diff
@@ -356,5 +356,5 @@ func handleDisconnectNotification(c *rpc2.Client) {
 
 // Disconnect will close the OVSDB connection
 func (ovs OvsdbClient) Disconnect() {
-	ovs.rpcClient.Close()
+	_ = ovs.rpcClient.Close()
```

Comment:
may be return error in case of error?

Reply:
@vtolstov the fix in this PR is to just silence my IDE in complaining whenever I try to `commit` changes. I will submit a separate PR to handle the error case. Thanks.

Reply:
as the fix is simple, i'm prefer to get it with this pr. or if you don't have time - revert changes that silence vet tools

Reply:
@vtolstov this PR was not meant for that, so I am not sure what is the resistance to do this in the Phase manner.